foladipo/docs-cabinet-cp2
# Implement all remaining feedback

#### What does this PR do?

This PR implements all the remaining feedback received from the (effective) Product Owner of this project before my final delivery. 

#### Description of task to be completed

Implement outstanding feedback.

#### How should this be manually tested?

Check the list of all the feedback that I should have addressed as listed in [this Pivotal Tracker story](https://www.pivotaltracker.com/n/projects/2034565/stories/148530777) and confirm that I have addressed them.


#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148530777
